assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -19,4 +18,17 @@ export const tests: Tests = {
     seed: 3,
   },
 
+  ssr: {
+    variants: ['off', 'on'],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 0,
```

Comment:
this is switched on but noone will be automatically allocated, it will all happen via the url parameters